src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2

```diff
@@ -0,0 +1,299 @@
+ASF Sentinel-1 Burst InSAR Data Package (ISCE2)
+===============================================
+
+This folder contains burst-based SAR Interferometry (InSAR) products and their associated files. The source data for
+these products are Sentinel-1 bursts, extracted from Single Look Complex (SLC) products processed by ESA,
+and they were processed using InSAR Scientific Computing Environment version 2 (ISCE2) software.
+
+Refer to https://sentinels.copernicus.eu/web/sentinel/user-guides/sentinel-1-sar/acquisition-modes/interferometric-wide-swath
+for more information on Sentinel-1 bursts.
+
+This data was processed by ASF DAAC HyP3 {{ processing_date.year }} using the {{ plugin_name }} plugin version
+{{ plugin_version }} running {{ processor_name }} release {{ processor_version }}.
+Files are projected to {{ projection }}, and the pixel spacing is {{ pixel_spacing|int }} m.
+
+The source bursts for this InSAR product are:
+ - Reference: {{ reference_granule_burst_name }}
+ - Secondary: {{ secondary_granule_burst_name }}
+
+Processing Date/Time: {{ processing_date.isoformat(timespec='seconds') }}
+
+The product directory is named <reference_burst_name>x<secondary_burst_name>
+
+The directory name for this product is: {{ reference_granule_burst_name }}x{{ secondary_granule_burst_name }}
+
+The reference_burst_name and secondary_burst_name use the following naming conventions:
+
+S1_bbbbbb_xxn_yyyymmddThhmmss_pp_cccc-BURST
+
+bbbbbb: Relative burst ID values assigned by ESA. Each value identifies a consistent burst footprint; relative burst ID values differ from one subswath to the next.
+
+xxn: Three character combination that represents the acquisition mode and subswath number. The first two characters xx represent the acquisition mode (IW or EW) and the last character identifies the subswath number (1-3 for IW, 1-5 for EW). IW mode indicates Interferometric Wideswath, which acquires a 250 km swath composed of three subswaths. EW mode indicates Extra-Wide Swath, which acquires a 400 km swath composed of 5 subswaths.
```

Comment:
I edited the description and left out the resolution info for the acquisition modes. I almost left out the IW/EW descriptions (last two sentences), but left them for now. How detailed do we want to get here? Does the ability to process EW granules impact the way that the pixel spacing is calculated? All of our original documentation was geared for IW; thought I'd check to make sure there isn't unexpected fallout.

Reply:
I'd like to keep it in.